### ISSUE TITLE
Having issues moving videos directory

### ISSUE THREAD

**Issue by gregk63** [WWBN/AVideo#4074]
I am trying to move the video directory to a NAS device. I purchased the SecureDirectory plugin and I followed the instructions to move everything. I am seeing I might have mounted it incorrectly because I can't change ownership to www-data. The command takes but ls shows it still has root ownership. I only noticed that it didn't actually change until I executed the mv command and things broke with permission problems. 

So I now I am trying to unmount to try and mount it again changing mount command but the unmount is failing saying it is busy. I tried disabling the plugin but it won't thinking that what was keeping the mnt busy.

Is there something I am missing that needs to happen to do the unmount?

Thanks,

Greg

**Comment by gregk63** [comment]
Is this something I could pay to have help with?

Thanks,

Greg

**Comment by DanielnetoDotCom** [comment]
I am not a server expert.

I can help you with AVideo software but not with your server configuration.

**Comment by akhilleusuggo** [comment]
@gregk63 go to where you want to change the owner directory ( root public apache directory , default /var/www/html )
Execute , `sudo chown www-data:www-data -R * `
then list to verify , `ls -la`
`chmod` if you need to change permissions ( read,write,execute ) . Google for more info .

**Comment by gregk63** [comment]
Hello akhilleusuggo, 

Thanks for the response. The directory I am trying to move to is a Network Attached drive. The mount seems to be fine. I am able to see other files on it but when I try the chown to change ownership to www-data as per the instructions it doesn't error but when you do an ls -la root still shows as owner. I went ahead and finished the instructions to move to this new directory and I received a bunch of permissions errors from the AVideo software. It seems it really needs the directory videos owned by www-data.

Any more assistance you can provide would be greatly appreciated.

Thanks,

Greg

**Comment by gregk63** [comment]
Thanks for your response Daniel!

**Comment by gregk63** [comment]
I resolved the permission issues and move everything to my NAS device. It all seemed to work but none of the pages were working. I grabbed some screen shots and move it back. 

I have attached the two screen shot of the pages that weren't working and the avideo.log.

Did I miss something?

![attempt to login](https://user-images.githubusercontent.com/72754374/102501670-4d5f2680-4043-11eb-88fa-6e41b4e90076.png)
![403 page](https://user-images.githubusercontent.com/72754374/102501727-5fd96000-4043-11eb-8c1f-f8edf0705290.png)
[avideo.log](https://github.com/WWBN/AVideo/files/5710113/avideo.log)
